compiler/control/OptimizationPlan.cpp

```diff
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include "env/TRMemory.hpp"
 #include "infra/Monitor.hpp"
+#include "control/CompilationController.hpp"
```

Comment:
Since the controller is now in a separate file, maybe it makes sense to eliminate this include and the lines that say
```
 if (TR::CompilationController::verbose() >= TR::CompilationController::LEVEL1)
   fprintf(stderr, "TR_OptimizationPlan allocations=%lu releases=%lu\n", _numAllocOp, _numFreeOp);
```
We haven't looked at this diagnostic output in a very long time.

Reply:
Pls check [067d872](https://github.com/eclipse/omr/commit/067d872acec8daf3056f3919a399d6b3bb46f07d)